cmd/nfd-master/main.go

```diff
@@ -135,6 +135,8 @@ func initFlags(flagset *flag.FlagSet) (*master.Args, *master.ConfigOverrideArgs)
 	flagset.BoolVar(&args.EnableNodeFeatureApi, "enable-nodefeature-api", true,
 		"Enable the NodeFeature CRD API for receiving node features. This will automatically disable the gRPC communication."+
 			" DEPRECATED: will be removed in a future release along with the deprecated gRPC API.")
+	flagset.BoolVar(&args.EnableNodeFeatureGroupApi, "enable-nodefeaturegroup-api", false,
+		"Enable the NodeFeatureGroup CRD API for receiving cluster feature rules.")
```

Comment:
I think is better to go for `alpha=disabled-by-default and turn it on in the next release.`

Reply:
>Another consideration is that should we introduce smth like kubernetes feature gates (like -feature-gates NodeFeatureGroupApi=true) to control this kind of new features instead of a somewhat arbitrary -enable-x or -disable-y flags. WDYT?

This sounds like a nice followup PR on it's own

Reply:
The more I look at this I think we should just add the `-feature-gates` flag and make this the first user of it. Either add it in a separate PR or this one. WDYT?